app/src/main/java/com/kelvin/bdaypost/login/data/AuthenticationRepository.kt

```diff
@@ -43,8 +43,10 @@ class AuthenticationRepository private constructor(
         @Volatile
         private var instance: AuthenticationRepository? = null
 
-        fun getInstance() = instance ?: synchronized(this) {
-            instance ?: AuthenticationRepository().also {
+        fun getInstance() = instance
```

Comment:
My Android Studio kept auto-formatting to this 🤷

Reply:
yaaa mine too, weird